preliz/distributions/continuous.py

```diff
@@ -1042,8 +1042,99 @@ def _fit_mle(self, sample, **kwargs):
         self._update(mu, sigma, nu)
 
 
+class TruncatedNormal(Continuous):
+    r"""
+    TruncatedNormal distribution.
+
+    The pdf of this distribution is
+
+    .. math::
+
+       f(x;\mu ,\sigma ,a,b)={\frac {\phi ({\frac {x-\mu }{\sigma }})}{
+            \sigma \left(\Phi ({\frac {b-\mu }{\sigma }})-\Phi ({\frac {a-\mu }{\sigma }})\right)}}
+
+    .. plot::
+        :context: close-figs
+
+        import arviz as az
+        from preliz import TruncatedNormal
+        az.style.use('arviz-white')
+        mus = [0.,  0., 0.]
+        sigmas = [3.,5.,7.]
+        lowers = [-3, -5, -5]
+        uppers = [7, 5, 4]
+        for mu, sigma, lower, upper in zip(mus, sigmas,lowers,uppers):
+            TruncatedNormal(mu, sigma, lower, upper).plot_pdf(support=(-10,10))
```

Comment:
nitpicking, we don't need to specify the support for this distribution

```suggestion
            TruncatedNormal(mu, sigma, lower, upper).plot_pdf()
```